test_main_objs.py

```diff
@@ -107,10 +135,13 @@ def test_get_sounding_count():
         for row in hardcoded_sc.index:
             assert hardcoded_sc.loc[row] == sc.loc[row]
 
+
 def interval_settings_helper(df, hardcoded_df):
```

Comment:
Does pytest require all unit tests to start with the word "test"? Also could there be a doc string for this?